test/js-api/exception/jsTag.tentative.any.js

```diff
@@ -0,0 +1,122 @@
+// META: global=window,dedicatedworker,jsshell,shadowrealm
+// META: script=/wasm/jsapi/assertions.js
+// META: script=/wasm/jsapi/wasm-module-builder.js
+
+test(() => {
+  assert_throws_js(TypeError, () => new WebAssembly.Exception(WebAssembly.JSTag, [{}]))
+}, "Creating a WebAssembly.Exception with JSTag explicitly is not allowed");
```

Comment:
Here I'm showing my lack of knowledge about the details of the test infra, but does this assertion check the exact text of the error message? The spec mandates what kind of exception should be thrown (TypeError), but not the text.

Reply:
I don't think this string is checked against anything else. So I just used it to describe what the tests were about. Maybe I'm abusing it? 🤷🏻

Reply:
Yeah, I think it's fine as long as the test won't fail because of different error messages.

Reply:
Yeah, this is just the name of the subtest.